docs/OMF.rst

```diff
@@ -236,33 +315,155 @@ The *equal* and *notequal* tests take a object as a child, the name of the objec
             }
       }
 
-In this case if the asset has a data point called *room* with a value of *4* then the asset will be placed in the AF location *ElectricalLab*, if it has a value of *6* then it is placed in the AF location *FluidLab*.
+In this case if the asset has a data point called *room* with a value
+of *4* then the asset will be placed in the AF location *ElectricalLab*,
+if it has a value of *6* then it is placed in the AF location *FluidLab*.
+
+If an asset matches multiple rules in the ruleset it will appear in
+multiple locations in the hierarchy, the data is shared between each of
+the locations.
+
+If an OMF Hint exists within a particular reading this will take
+precedence over generic rules.
+
+The AF location may be a simple string or it may also include
+substitutions from other data points within the reading. For example
+of the reading has a data point called *room* that contains the room
+in which the readings was taken, an AF location of */BuildingA/${room}*
+would put the reading in the asset framework using the value of the room
+data point. The reading
+
+.. code-block:: JSON
+
+  "reading" : {
+       "temperature" : 23.4,
+       "room"        : "B114"
+       }
```

Comment:
Could not lex literal block as "JSON". So, it's highlighting will be skipped. 
If we really want to see as a JSON then we need to fix the JSON format by adding open and close braces {}. otherwise we can simply add code-block:: console

Reply:
We use "console" elsewhere, so maybe this is the best approach